nodehid.js

```diff
@@ -12,18 +9,14 @@ function setDriverType(type) {
 // lazy load the C++ binding
 var binding = null;
 function loadBinding() {
-    if( !binding ) {
-        if( os.platform() === 'linux' ) {
-            // Linux defaults to hidraw
-            if( !driverType || driverType === 'hidraw' ) {
-                binding = require('bindings')('HID_hidraw.node');
-            } else {
-                binding = require('bindings')('HID.node');
+    if (!binding) {
+        binding = require("pkg-prebuilds/bindings")(
+            __dirname,
+            {
```

Comment:
The purpose of `binding-options.js` is to avoid repetition of the same configuration, so it would be better to build this object with something like:
```
const options = require('./binding-options')
if (process.platform === "linux" && (!driverType || driverType === "hidraw")) options.name = 'HID_hidraw'
binding = require("pkg-prebuilds/bindings")(__dirname, options)
```

That way, when an option wants to be set/changed, it only has to be done in one place